src/cli/usage.js

```diff
@@ -52,7 +49,12 @@ ${bold('OPTIONS')}
 		Display application changelog.
 	
 	--accept-self-signed-certs
-		Allows medic-conf to work with self signed certs by telling node to ignore the error 
+    Allows medic-conf to work with self signed certs by telling node to ignore the error
+    
+${bold('SUPPORTED ACTIONS')}
+  Include a list of supported actions after '--':
```

Comment:
Is this true? I thought the supported actions were used *before* the `--`, and the `--` was used to signify that you wanted to pass params to an action you were invoking?

So like: `medic-conf do-a-thing -- some-param`

Reply:
Definitely messed this up.